File: server/models/task.js
const Sequelize = require('sequelize');
module.exports = function (sequelize) {
  const task = sequelize.define('task', {
    description: {
      type: Sequelize.STRING,
      field: 'description'
    },
    time: {
      type: Sequelize.INTEGER,
      field: 'time'
    },
    synced: {
      type: Sequelize.BOOLEAN,
      field: 'synced'
    }
  });

  return task;
};

File: server_native/src/models/task.rs
#[derive(Debug)]
#[derive(Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct Task  {
    pub id: Option<i64>,
    pub description: Option<String>,
    pub time: Option<f64>,
    pub created_at: Option<String>,
    pub updated_at: Option<String>,
    pub day_id: Option<i64>,
    pub synced: Option<bool>
}